index.html

```diff
@@ -260,7 +263,42 @@ <h1 class="section-title">
                 </div>
             </section>
             <!-- /PAGE ABOUT -->
-
+            <!-- PAGE DIVERSITY -->
+            <section class="page-section" id="diversity">
+                <div class="container">
+                    <h1 class="section-title">
+                        <span data-animation="flipInY" data-animation-delay="300" class="icon-inner"><span
+                                class="fa-stack"><i class="fa rhex fa-stack-2x"></i><i
+                                class="fa fa-star fa-stack-1x"></i></span></span>
+                        <span data-animation="fadeInRight" data-animation-delay="500" class="title-inner">Diversity Matters <small>/ We welcome you</small></span>
+                    </h1>
+                    <div class="row">
+                        <div class="col-lg-8">
+                            <p data-animation="fadeInUp" data-animation-delay="300">
+                                For those of you who feel part of an underrepresented group in tech, we welcome
+                                you.
+                            </p>
+                            <p>
+                                We want to support you by reserving tickets for 50 people on the first
+                                ticket sale wave. This includes (but is not limited to): people of
+                                colour, disabled people, LGBTQIA+ people and women.
+                            </p>
+                            <p>
+                                This year we will release tickets in three different waves:
```

Comment:
Maybe we could split this information into its own "Ticket sales" section. Since it's some kind of crucial information, it could be interesting to do not hide it inside the Diversity section. What do you think?